recipes-kernel/linux/files/prplmesh.cfg

```diff
@@ -0,0 +1,28 @@
+CONFIG_TUN=y
```

Comment:
We don't use tunneling, so I wonder why this is here?

Reply:
[CONFIG_TUN](https://cateee.net/lkddb/web-lkddb/TUN.html) is option not for tunneling but for "Universal TUN/TAP device driver support"

Reply:
Yes I know. But we don't use tun/tap in prplmesh. Or is it something you need for qemu?